package.json

```diff
@@ -273,6 +273,7 @@
   },
   "devDependencies": {
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@graphql-inspector/core": "^1.14.0",
```

Comment:
I added this to Peril a few weeks back, it provides a way for us to take the generic changes between two GraphQL schemas and makes human readable messages from it.